easybuild/easyblocks/f/fftw.py

```diff
@@ -28,15 +28,16 @@
 @author: Kenneth Hoste (HPC-UGent)
 """
 from distutils.version import LooseVersion
-from vsc.utils.missing import nub
 
 from easybuild.easyblocks.generic.configuremake import ConfigureMake
 from easybuild.framework.easyconfig import CUSTOM
 from easybuild.toolchains.compiler.gcc import TC_CONSTANT_GCC
+from easybuild.tools.build_log import EasyBuildError
```

Comment:
It didn't work, the import was always missing, it's just that we didn't notice.

Perhaps this should be a separate PR, since it's a silly bug that's also relevant for current `master` & `develop`. I'll do exactly that.

Reply:
fixed in #1641